src/EventTeachingService.cpp

```diff
@@ -11,259 +11,427 @@
 
 namespace VLCB {
 
-void EventTeachingService::setController(Controller *cntrl)
+void EventTeachingService::setController(Controller *cntrl) 
 {
   this->controller = cntrl;
   this->module_config = cntrl->module_config;
 }
 
-void EventTeachingService::enableLearn()
+void EventTeachingService::enableLearn() 
 {
   bLearn = true;
-   //DEBUG_SERIAL << F("> set learn mode ok") << endl;
+  DEBUG_SERIAL << F("> set learn mode ok") << endl;
   // set bit 5 in parameter 8
   bitSet(controller->_mparams[8], 5);
 }
 
-void EventTeachingService::inhibitLearn()
+void EventTeachingService::inhibitLearn() 
 {
   bLearn = false;
-   //DEBUG_SERIAL << F("> learn mode off") << endl;
+  DEBUG_SERIAL << F("> learn mode off") << endl;
   // clear bit 5 in parameter 8
   bitClear(controller->_mparams[8], 5);
 }
 
-Processed EventTeachingService::handleMessage(unsigned int opc, CANFrame *msg)
-{
+Processed EventTeachingService::handleMessage(unsigned int opc, CANFrame *msg) {
```

Comment:
Please keep the curly-brace style were they are on new lines.

Reply:
Done